sda/conode.go

```diff
@@ -71,11 +84,22 @@ func (c *Conode) GetStatus() Status {
 	a := ServiceFactory.RegisteredServiceNames()
 	sort.Strings(a)
 	m["Available_Services"] = strings.Join(a, ",")
+	m["TX_bytes"] = strconv.FormatUint(c.Router.Tx(), 10)
+	m["RX_bytes"] = strconv.FormatUint(c.Router.Rx(), 10)
+	m["Uptime"] = time.Now().Sub(c.started).String()
+	m["System"] = fmt.Sprintf("%s/%s/%s", runtime.GOOS, runtime.GOARCH,
+		runtime.Version())
+	m["Version"] = "0.9.1"
```

Comment:
did you not put a `const version` string somewhere in the code ? Maybe I dreamt it but I thought I saw something like that hanging around.